src/utils/ProfileUtils.ts

```diff
@@ -89,10 +75,12 @@ export async function installModsToProfile(
     const installedVersions = profileMods.map((m) => m.getDependencyString());
     const disabledMods = disabledModsOverride || profileMods.filter((m) => !m.isEnabled()).map((m) => m.getName());
     let currentMod;
```

Comment:
After the changes currentMod isn't really used and there's no point tracking the same thing twice, so might as well yeet it.

I do like the empty line before try-block though, so please don't yeet it.